src/core/Table/Table.tsx

```diff
@@ -149,6 +170,9 @@ export const Table = <
     onBottomReached,
     onRowInViewport,
     intersectionMargin = 300,
+    expandedSubComponent,
+    onExpand,
+    provideDefaultExpander: provideExpanderColumn = true,
```

Comment:
New variable looks weird here.. Why you needed it?

Reply:
Oh.. this is some weird F2 renaming thing in Vscode. When I renamed prop name, it preserved older name and put it after colon... Will fix it

Reply:
Fixed with new commits